test/browser_tests/page_objects/page_bureau-structure.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+function TheBureauStructurePage() {
+
+  this.get = function( ) {
+    browser.get( '/the-bureau/bureau-structure/' );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+
+  this.sideNav = element( by.css( '.nav-secondary' ) );
```

Comment:
Should the variable name be consistent with the class? E.g.:

`this.navSecondary = element( by.css( '.nav-secondary' ) );`